js/parallax.js

```diff
@@ -156,7 +161,8 @@ const asteroidProto = {
                             [
                                 this.x + this.width / 2 + i * (this.size - 1) * 25, this.y,
                                 -this.dx / 2 + randomFloat(-0.75, 1.5),
-                                -this.dy / 2 + randomFloat(-0.75, 1.5)
+                                -this.dy / 2 + randomFloat(-0.75, 1.5),
+                                Math.max(-4, Math.min(4, this.rotPerFrame + randomFloat(-2.5, 5))) // == clamp into [-4, 4]

```

Comment:
now this line confuses me. I don't understand why you're getting a random float up to 5, only to cap the result at 4.
Also, could a `clamp(...)` helper function be useful, similar to the random generators?

Reply:
Ideally place magic numbers in constants with nice clear names to help avoid such confusion 👍

Reply:
I am not getting random number up to 5, I am getting random number in range `[-2.5, 2.5]` ;)